### PR TITLE
Adding scanning guide

### PR DESCRIPTION
**Pull request by mick-roper**
* added a scanning guide
* copied `install` into `guides`
* copied `org management` into `guides`

### CODE REVIEW

File: docs/getting-started/scanning-k8s/index.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: Loging in to Reliably
+excerpt: Login to Reliably with the Reliably CLI to monitor SLOs, scans Kubernetes clusters and manifests and collaborate with teammates.
```
Comment (Lawouach) [inline docs/getting-started/scanning-k8s/index.md:3]: change these?

File: docs/getting-started/scanning-k8s/index.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+---
+
+# Scanning Kubernetes for potential problems
+
+Reliably offers the ability to prevent some issues that can occur in incorrecttly deployed kubernetes applications. The `reliably scan` command is the tool used to scan your infrastructure and deployments for these issues.
+
+## Before you get started
+
+* [install the Reliably CLI](../../getting-started/install/index.md)
+* [create an organisation](../../getting-started/org-management/index.md)
```
Comment (Lawouach) [inline docs/getting-started/scanning-k8s/index.md:16]: this is not strictly needed in a 5mn guide to be fair. You already have your own org.

File: docs/getting-started/scanning-k8s/index.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+## Whats 'scanning'?
+
+Reliably integrates with [Open Policy Agent](https://github.com/open-policy-agent/opa) to offer the ability to see divergence from best practice in your applications. Adhering to these 'policies' gives a sense of confidence that your application is making use of best practices, and potentially eliminating some issues before they cause trouble - here at Reliably we believe a problem avoided is better than a problem solved!
```
Comment (Lawouach) [inline docs/getting-started/scanning-k8s/index.md:20]: OPA is an implementation detail. We used to hide it. Not sure there is a need for the user to know that.

File: docs/getting-started/scanning-k8s/index.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+### Reliably and OPA
+
+So why not just use OPA directly? The answer is a little strange - OPA's most powerful feature is also one of its most complex components. The policies are simultaneously an incredible tool, while also being difficult to write and maintain.
+
+The integration with Reliably means that ReliablyHQ can:
+* offer policies that deal with low-hanging fruit, by using our knowledge of running applications 'in the wild' to give you an excellent starting point.
+* dynamically update your policies based on what we see happening in your infrastructure.
```
Comment (Lawouach) [inline docs/getting-started/scanning-k8s/index.md:28]: totally, but simply remove the mention to OPA and it's just as clear that we provide production ready-made policies :)

File: docs/guides/install/index.md [hunk 5]
```diff
@@ -0,0 +1,323 @@
+---
```
Comment (Lawouach) [inline docs/guides/install/index.md:1]: Please do not confuse commits. Make the change of that page location in a specific PR.

File: docs/guides/org-management/index.md [hunk 6]
```diff
@@ -0,0 +1,166 @@
+---
```
Comment (Lawouach) [inline docs/guides/org-management/index.md:1]: likewise, have a specific commit and PR for this move.

**Review by Lawouach — CHANGES_REQUESTED** [review]
Move the pages in separated commits and, even better, separated PRs please.

File: docs/getting-started/scanning-k8s/index.md (hunk 3 above)
Comment (mick-roper) [reply docs/getting-started/scanning-k8s/index.md:20]: Its our only implementation - and its in a CLI that anyone can see if they look. is there an upside to obfuscating the fact that we use open-source software in our stack?

File: docs/getting-started/scanning-k8s/index.md (hunk 4 above)
Comment (mick-roper) [reply docs/getting-started/scanning-k8s/index.md:28]: Response as above

File: docs/guides/install/index.md (hunk 5 above)
Comment (mick-roper) [reply docs/guides/install/index.md:1]: OK

File: docs/guides/org-management/index.md (hunk 6 above)
Comment (mick-roper) [reply docs/guides/org-management/index.md:1]: OK

File: docs/getting-started/scanning-k8s/index.md (hunk 2 above)
Comment (mick-roper) [reply docs/getting-started/scanning-k8s/index.md:16]: removed from the document

File: docs/getting-started/scanning-k8s/index.md (hunk 1 above)
Comment (mick-roper) [reply docs/getting-started/scanning-k8s/index.md]: done

File: docs/getting-started/scanning-k8s/index.md (hunk 3 above)
Comment (Lawouach) [reply docs/getting-started/scanning-k8s/index.md:20]: Not sure there is one to mention it when that's an implementation detail. It'd make sense to mention it if people could write their own policies, it's not the case for now.

Comment (Lawouach) [reply docs/getting-started/scanning-k8s/index.md:20]: Feel free to keep as you wrote down, it's likely harmless.

Comment (mick-roper) [reply docs/getting-started/scanning-k8s/index.md:20]: OK i'll update the docs

**Review by Lawouach — APPROVED** [review]

**Pull request merged**